src/sentry/api/endpoints/group_details.py

```diff
@@ -263,6 +267,35 @@ def get(self, request, group):
             }
         )
 
+        # the current release is the 'latest seen' release within the
+        # environment even if it hasnt affected this issue
+
+        try:
+            environment = self._get_environment_from_request(
+                request,
+                group.project.organization_id,
+            )
+        except Environment.DoesNotExist:
+            environment = None
+
+        if environment is not None:
+            current_release = GroupRelease.objects.filter(
+                group_id=group.id,
+                environment=environment.name,
+                release_id=ReleaseEnvironment.objects.filter(
+                    release_id__in=ReleaseProject.objects.filter(project_id=group.project_id
+                                                                 ).values_list('release_id', flat=True),
+                    organization_id=group.project.organization_id,
+                    environment_id=environment.id,
+                ).order_by('-first_seen').values_list('release_id', flat=True).first(),
+            ).first()
```

Comment:
mm i remember @mattrobenolt mentioning that we should use `[0]` and handle the index error rather than `first` if we're not defining an `order_by` clause, so i think the use on 290 is ok, but you should probably change 291. @mattrobenolt can you confirm/provide a better explanation as to why?

Reply:
That's correct, but there is an ORDER BY clause in here, so it'd be fine. Generally, I'd just prefer to see us get away from using `.first()` without thinking about it since it does change the behavior in usually unintended ways and sometimes with bad side effects..

Reply:
Oh, I actually didn't read close enough in the actual PR.

So this `.first()` being pointed out, you're right. This one doesn't have an ORDER BY. There are a few nested things going on here.

The _other_ use of `.first()` is actually being evaluated first, and not translating into a subquery, since `.first()` doesn't return a QuerySet, nor would `[0]`. In this nested queryset, we should use `[0:1]` or `[:1]` to get the actual nested query that behaves how you want.